.gitignore

```diff
@@ -51,3 +54,20 @@ modules
 run-tests.php
 run-tests.log
 tmp-php.ini
+
+# Windows build files
+x64
+x86
```

Comment:
There is no x86 folder for x86 builds. Instead Debug(_TS)/Release(_TS) are created in the root directory.